content/en/continuous_integration/pipelines/_index.md

```diff
@@ -24,7 +24,23 @@ cascade:
 
 ## Overview
 
-[Pipeline Visibility][1] provides a pipeline-first view into your CI health by displaying important metrics and results from your pipelines. It can help you investigate performance problems and test failures that concern you the most because you work on the related code, not because you maintain the pipelines they are run in.
+[Pipeline Visibility][1] provides a pipeline-first view into your CI health by displaying important metrics and results from your pipelines. It helps you troubleshoot pipeline failures, address performance bottlenecks, and track CI performance and reliability over time.
+
+### Supported features
+
+|  | Jenkins | GitLab | CircleCI | Buildkite | GitHub Actions | Azure Pipelines | Codefresh | TeamCity |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| {{< ci-details title="Pipeline trace visualization" >}}The ability to do pipeline execution visualization with associated tracing.{{< /ci-details >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} |
+| {{< ci-details title="Partial retries" >}}The ability to identify partial retries (for example, when only a subset of jobs were retried) in CI providers.{{< /ci-details >}} |  | {{< X >}} |  | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} |
+| {{< ci-details title="Manual steps" >}}The ability to identify when there is a job with a manual approval phase in the overall pipeline, which is split into two pipelines: <br>- From the beginning of the pipeline to when the manual approval action starts and the pipeline becomes <code>Blocked</code>.<br>- From the end of the manual approval action to the end of the pipeline.<br>{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  | {{< X >}} |  |
+| {{< ci-details title="Queue time" >}}The ability to add the time for which a job (or pipeline depending on the CI provider) was in the queue before it executes.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |
+| {{< ci-details title="Logs correlation" >}}The ability to retrieve pipeline or job logs from the CI provider. Logs are displayed on the <strong>Logs</strong> tab in the Pipeline Execution view.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  |  | {{< X >}} |  |  |  |
+| {{< ci-details title="Infrastructure metric correlation" >}}The ability to display host-level information for the Datadog Agent, pipelines, or job runners.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  |  |  |
```

Comment:
Correlation of host-level information for the Datadog Agent, CI pipelines, or job runners to CI pipeline execution data